chart/templates/_config.tpl

```diff
@@ -46,7 +46,14 @@
 
     {{- $configs := dict }}
     {{- range $name, $bytes := $.Files.Glob "config/*" }}
-      {{- $_ := set $configs $name ($bytes | toString | fromYaml) }}
+      {{/*
+      **** !Attention! Templates in config files have only access to the global helm variables,
```

Comment:
I was thinking that you would add this to the documentation of this function at the top, and not in the middle of the function): explain that config files are evaluated as templates, but that the available data is limited.

Reply:
Will change

Reply:
Done: 4b1b0cf0e199ac06bc5bc9dddef10e3bf29f30a0